src/components/Camera/index.js

```diff
@@ -76,10 +78,10 @@ export default class Camera extends Component {
     const { method, onUserMedia } = this.props
     return (
       <div>
-        <div className='onfido-video-overlay'>
+        <div className={style["video-overlay"]}>
```

Comment:
Just because of [the style guide](https://github.com/airbnb/javascript/tree/master/react#quotes)